sn_cli/src/subcommands/register.rs

```diff
@@ -41,43 +52,44 @@ pub(crate) async fn register_cmds(
 ) -> Result<()> {
     match cmds {
         RegisterCmds::Create { name } => create_register(name, client, verify_store).await?,
-        RegisterCmds::Edit { name, entry } => {
-            edit_register(name, entry, client, verify_store).await?
-        }
-        RegisterCmds::Get { names } => get_registers(names, client).await?,
+        RegisterCmds::Edit {
+            address,
+            use_name,
+            entry,
+        } => edit_register(address, use_name, entry, client, verify_store).await?,
+        RegisterCmds::Get {
+            addresses,
+            use_name,
+        } => get_registers(addresses, use_name, client).await?,
     }
     Ok(())
 }
 
 async fn create_register(name: String, client: &Client, verify_store: bool) -> Result<()> {
-    let tag = 3006;
-    let xorname = XorName::from_content(name.as_bytes());
-    println!("Creating Register with '{name}' at xorname: {xorname:x} and tag {tag}");
-
-    // clients currently only support public registers as we create a new key at each run
-    let _register =
-        ClientRegister::create_public_online(client.clone(), xorname, tag, verify_store).await?;
-    println!("Successfully created register '{name}' at {xorname:?}, {tag}!");
+    let meta = XorName::from_content(name.as_bytes());
+    let register = ClientRegister::create_online(client.clone(), meta, verify_store).await?;
+    println!(
+        "Successfully created register '{name}' at {}!",
+        register.address()
+    );
     Ok(())
 }
 
 async fn edit_register(
-    name: String,
+    address: String,
```

Comment:
`address` indicates it's a XorName or something relevant (say HexString of Key)
better consider use other var name, or at least put a comment to explain what is this String for address?